runtime/src/main/java/org/corfudb/runtime/view/SealServersHelper.java

```diff
@@ -86,7 +87,9 @@ public static void waitForChainSegmentSeal(Layout.LayoutSegment layoutSegment, M
             boolean success = false;
             try {
                 success = quorumFuture.get();
-            } catch (ExecutionException | InterruptedException e) {
+            } catch (InterruptedException ie) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)